src/context/actions/types.tsx

```diff
@@ -9,9 +9,11 @@ export type ActionMap<M extends { [index: string]: any }> = {
   }
 };
 
+
 export enum Types {
   Login_update = 'LOGIN_UPDATE',
-  Logout_update = 'LOGOUT_UPDATE'
+  Logout_update = 'LOGOUT_UPDATE',
+  NotificationActionTypes = "NotificationActionTypes"
```

Comment:
It seems like `NotificationActionTypes` is unused. Please, verify if it is actually used, if not, then remove it. Thanks

Reply:
Yup, unused. Removed it. Great catch!